routes/error-tracker.js

```diff
@@ -209,11 +212,17 @@ function getHandler(req, res) {
       },
     },
   };
-  // get authentication context for logging
-  const loggingClient = logging({
-    projectId: appEngineProjectId,
-  });
-  const log = loggingClient.log(logName);
+  if (params.debug === '1' ) {
+    res.set('Content-Type', 'application/json; charset=ISO-8859-1');
+    res.status(statusCodes.OK).send(
+        JSON.stringify({
+          message: 'OK\n',
+          event: event,
+          throttleRate: throttleRate,
```

Comment:
I meant that you lost the previous changes. Specifically, use of property short-form: `{message: 'OK\n', event, throttleRate}`